service/history/timerQueueStandbyProcessor.go

```diff
@@ -237,6 +241,25 @@ func (t *timerQueueStandbyProcessorImpl) processDecisionTimeout(timerTask *persi
 	})
 }
 
+func (t *timerQueueStandbyProcessorImpl) processRetryTimer(timerTask *persistence.TimerTaskInfo) error {
```

Comment:
@wxing1292 can you make sure this looks good from passive processing of retry tasks?

Reply:
@samarabbas 
the retry timer task, should not be processed by the standby timer processor, since only the active cluster get to process the activity retry. standby cluster will only deal with "existing" history events, so, the "phantom" events created by active cluster will not be replicated to standby cluster, making retry timer meaningless to standby processor.